app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt

```diff
@@ -76,6 +77,11 @@ fun Context.setIsLoggedIn(isLoggedIn: Boolean) =
 fun Context.isLoggedIn(): Boolean =
     getSharedPreferences().getBoolean(SHARED_PREFERENCE_IS_LOGGED_IN, false)
 
+fun Context.getMarket(): Market {
+    val marketId = getSharedPreferences().getInt(Market.MARKET_SHARED_PREF, -1)
```

Comment:
`-1` is probably unsafe as a default value. Perhaps this function should return `Market?` and provide null if no market is set?